src/template-tags/tickets.php

```diff
@@ -1727,3 +1727,36 @@ class_exists( $provider )
 		);
 	}
 }
+
+/**
+ * Determine whether new tickets (block) views are enabled.
+ *
+ * In order the function will check the `TRIBE_TICKETS_NEW_VIEWS` constant,
+ * the `TRIBE_TICKETS_NEW_VIEWS` environment variable and, finally, the `tribe_tickets_new_views_is_enabled` option.
+ *
+ * @since TBD
+ *
+ * @return boolean Whether new tickets views are enabled.
+ */
+function tribe_tickets_new_views_is_enabled() {
+	// Check for constant.
+	if ( defined( 'TRIBE_TICKETS_NEW_VIEWS' ) ) {
+		return (boolean) TRIBE_TICKETS_NEW_VIEWS;
```

Comment:
:no_entry_sign: Short form type keywords must be used. Found: (boolean)

<sub>hash: d26b4d5924eaf88bad49952c01890756</sub>